src/libexpr/eval.cc

```diff
@@ -1094,13 +1095,32 @@ void EvalState::callPrimOp(Value & fun, Value & arg, Value & v, const Pos & pos)
     }
 }
 
+void fnStart(const Pos & pos) {
+    if (verbosity >= lvlDebug) {
```

Comment:
Hm, actually it might be better to use `lvlVomit` given the amount of output that this is likely to produce...

Reply:
Done!

Reply:
Or activate it via a specific setting (e.g. `--trace-function-calls`).

Reply:
I would be happy to do that, but would need some pointers to how to do that.

Reply:
Add a flag to `EvalSettings` like
```
    Setting<bool> traceFunctionCalls{this, false, "trace-function-calls",
        "Bla bla..."};
```
And then do `if (evalSettings.traceFunctionCalls) ...`.